### ISSUE TITLE
nanotrack

### ISSUE THREAD

**Issue by czwWZC** [HonglinChu/SiamTrackers#163]
您好，这边nanotrack训练的时候   在data里的是事先裁剪好的训练图片吗

**Comment by HonglinChu** [comment]
yes, you can follow pysot https://github.com/STVIR/pysot/blob/master/TRAIN.md

**Comment by HonglinChu** [comment]
https://drive.google.com/file/d/1VwTAAiiT4gZVUVkoHSMV3vRJATrBSOIz/view?usp=drive_link  (GOT-10k)
